index.html

```diff
@@ -5,38 +5,56 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
-      rel="stylesheet"
-    />
     <title>Profile Rapheal Okonta</title>
   </head>
   <body>
     <header>
+     
       <nav class="nav_bar">
-        <a href="#" class="name_logo">RAPHEAL OKONTA</a>
-        <div class="link_header">
-        <div class="nav_header_link">
-          <a class="header_menu" href="#">Portfolio</a>
-          <a class="header_menu" href="#">About</a>
-          <a class="header_menu" href="#">Contact</a>
-          <img
+        <a href="#" class="nav_branding">RAPHEAL OKONTA</a>
+        <ul class="nav_menu">
+          <li class="nav_item">
+            <a href="#recent_works_link" class="nav_link">Portfolio</a>
+              <hr class="hr-line_break3">
+          </li>
+          <li class="nav_item">
+            <a href="#aboutmeProject" class="nav_link">About </a>
+          <hr class="hr-line_break3">
+                
+          </li>
+          <li class="nav_item">
+            <a href="#form_container_link" class="nav_link">contact</a>
+            <hr class="hr-line_break3">
```

Comment:
In order to make your design conform to that of the figma, kindly check and ensure tha the width, color and font-weight of the horizontal lines below the nav-links in the mobile menu matches those of the figma design. 

Your design             |  The figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-19 23-31-03](https://user-images.githubusercontent.com/65233860/196824892-2513e4ac-755d-49ad-83b9-b7644ca77b33.png)  |  ![Screenshot from 2022-10-19 23-31-35](https://user-images.githubusercontent.com/65233860/196824929-78af3945-b071-4b29-bbbd-e9ef95a4bacf.png)